src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -1117,9 +1117,10 @@ staticMethod org.codehaus.groovy.runtime.StringGroovyMethods replaceFirst java.l
 staticMethod org.codehaus.groovy.runtime.StringGroovyMethods reverse java.lang.String
 staticMethod org.codehaus.groovy.runtime.StringGroovyMethods takeWhile groovy.lang.GString groovy.lang.Closure
 staticMethod org.codehaus.groovy.runtime.StringGroovyMethods takeWhile java.lang.CharSequence groovy.lang.Closure
-staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toBoolean java.lang.Boolean
-staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toDouble java.lang.Double
-staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toInteger java.lang.Integer
+staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toBoolean java.lang.String
+staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toDouble java.lang.String
+staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toInteger java.lang.CharSequence
+staticMethod org.codehaus.groovy.runtime.StringGroovyMethods toInteger java.lang.String
```

Comment:
Ok one last thing, but `DefaultGroovyMethods.toInteger(String)` (which is deprecated) is (I think) the method that `SandboxInterceptor` will actually think you are calling if you write something like `"123".toInteger()`, so you probably need to add that one too. Regular Groovy does not consider deprecated methods when selecting extension methods, so I asked you to add the non-deprecated `StringGroovyMethods` equivalents in case we ever fix that bug 😅

Reply:
I still have that in the file, don't I?

Reply:
Ah, I missed that it was already present before your changes, so yeah it should be fine as-is.